nerve/check_https_test.go

```diff
@@ -0,0 +1,14 @@
+package nerve
+
+import (
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestHttpsDefault(t *testing.T) {
```

Comment:
with	"net/http/httptest", you should be able to test the check with an https call